docs: update getting-started config to match default generated config (#19308)
eslint/eslint: docs/src/use/getting-started.md

For example, one of the questions is "Where does your code run?" If you select "Browser" then your configuration file will contain the definitions for global variables found in web browsers. Here's an example:

```js
import globals from "globals";
import pluginJs from "@eslint/js";

/** @type {import('eslint').Linter.Config[]} */
export default [
  {languageOptions: { globals: globals.browser }},
  pluginJs.configs.recommended,
];
```

The `pluginJs.configs.recommended` object contains configuration to ensure that all of the rules marked as recommended on the [rules page](../rules) will be turned on.  Alternatively, you can use configurations that others have created by searching for "eslint-config" on [npmjs.com](https://www.npmjs.com/search?q=eslint-config).  ESLint will not lint your code unless you extend from a shared configuration or explicitly turn rules on in your configuration.

We can add `rules` as following:

```js
   import pluginJs from "@eslint/js";

   export default [
       pluginJs.configs.recommended,

      {
          rules: {
              "no-unused-vars": "warn",
              "no-undef": "warn"
          }
      }
   ];
```

> The names `"no-unused-vars"` and `"no-undef"` are the names of [rules](../rules) in ESLint. The first value is the error level of the rule and can be one of these values:

Reviewer: ```suggestion

The names `"no-unused-vars"` and `"no-undef"` are the names of [rules](../rules) in ESLint. The first value is the error level of the rule and can be one of these values:
```

we need an empty line here.

> - "off" or 0 - turn the rule off
> - "warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
> - "error" or 2 - turn the rule on as an error (exit code will be 1)

Reviewer: ```suggestion
* "off" or 0 - turn the rule off
* "warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
* "error" or 2 - turn the rule on as an error (exit code will be 1)
```

When you run `npm init @eslint/config`, you'll be asked a series of questions to determine how you're using ESLint and what options should be included. After answering these questions, you'll have an `eslint.config.js` (or `eslint.config.mjs`) file created in your directory.

For example, one of the questions is "Where does your code run?" If you select "Browser" then your configuration file will contain the definitions for global variables found in web browsers. Here's an example:

```js
import globals from "globals";
import pluginJs from "@eslint/js";

/** @type {import('eslint').Linter.Config[]} */
export default [
  {languageOptions: { globals: globals.browser }},
  pluginJs.configs.recommended,
];
```

The `pluginJs.configs.recommended` object contains configuration to ensure that all of the rules marked as recommended on the [rules page](../rules) will be turned on.  Alternatively, you can use configurations that others have created by searching for "eslint-config" on [npmjs.com](https://www.npmjs.com/search?q=eslint-config).  ESLint will not lint your code unless you extend from a shared configuration or explicitly turn rules on in your configuration.

> We can add `rules` as following:

Reviewer 2: ```suggestion
You can configure rules individually by defining a new object with a `rules` key, as in this example:
```

```js

>    import pluginJs from "@eslint/js";

Reviewer: ```suggestion
import pluginJs from "@eslint/js";
```

We can avoid extra indentation here.

>    export default [
>        pluginJs.configs.recommended,
> 
>       {
>           rules: {
>               "no-unused-vars": "warn",
>               "no-undef": "warn"
>           }
>       }
>    ];

Reviewer: ```suggestion
export default [
    pluginJs.configs.recommended,

    {
        rules: {
            "no-unused-vars": "warn",
            "no-undef": "warn"
        }
    }
];
```

here also.

**Note:** If you are coming from a version before 9.0.0 please see the [migration guide](configure/migration-guide).

> When you run `npm init @eslint/config` ,terminal would provide you different options to create a config file `eslint.config.js` (or `eslint.config.mjs`) as per your need.

Reviewer: ```suggestion
When you run `npm init @eslint/config`, terminal would provide you different options to create a config file `eslint.config.js` (or `eslint.config.mjs`) as per your need.
```

> When you run `npm init @eslint/config`, terminal would provide you different options to create a config file `eslint.config.js` (or `eslint.config.mjs`) as per your need.

Reviewer 2: To clarify what happens.
```suggestion
When you run `npm init @eslint/config`, you'll be asked a series of questions to determine how you're using ESLint and what options should be included. After answering these questions, you'll have an `eslint.config.js` (or `eslint.config.mjs`) file created in your directory.
```

> When you run `npm init @eslint/config`, you'll be asked a series of questions to determine how you're using ESLint and what options should be included. After answering these questions, you'll have an `eslint.config.js` (or `eslint-config.mjs`) file created in your directory.

Reviewer: ```suggestion
When you run `npm init @eslint/config`, you'll be asked a series of questions to determine how you're using ESLint and what options should be included. After answering these questions, you'll have an `eslint.config.js` (or `eslint.config.mjs`) file created in your directory.
```

When you run `npm init @eslint/config` ,terminal would provide you different options to create a config file `eslint.config.js` (or `eslint.config.mjs`) as per your need.

```js
// eslint.config.js
export default [
    {
        rules: {
            "no-unused-vars": "error",
            "no-undef": "error"
        }
    }
];
```

> The names `"no-unused-vars"` and `"no-undef"` are the names of [rules](../rules) in ESLint. The first value is the error level of the rule and can be one of these values:

Reviewer 2: I think we should keep this content as a next step, showing how to individually configure rules after the config file is generated for you.

Reviewer: I think it's good to keep it in configuration section just below we explained the `pluginJs.configs.recommended` we can write something like:
>Then we can configure rules as following:

and then the example configuration with rules, then the para explaining rules in example?

> For example if you select *browser` and choose to install the `dependencies` ,you will get a config like the one shown below.

Reviewer: ```suggestion
For example if you select *browser` and choose to install the `dependencies`, you will get a config like the one shown below.
```

> For example if you select *browser` and choose to install the `dependencies` ,you will get a config like the one shown below.

Reviewer 2: Fixing formatting.

```suggestion
For example if you select "browser" and choose to install the `dependencies` ,you will get a config like the one shown below.
```

> For example if you select *browser` and choose to install the `dependencies`, you will get a config like the one shown below.

Reviewer 2: For example, one of the questions is "Where does your code run?" If you select "Browser" then your configuration file will contain the definitions for global variables found in web browsers. Here's an example:

> For example if you select *browser* and choose to install the *dependencies* , you will get a config like the one shown below .

Author: done

> For example,one of the questions is "Where does your code run?" If you select "Browser" then your configuration file will contain the definitions for global variables found in web browsers. Here's an example .

Reviewer: ```suggestion
For example, one of the questions is "Where does your code run?" If you select "Browser" then your configuration file will contain the definitions for global variables found in web browsers. Here's an example .
```

Author: should I also include comma after Browser ?
"Browser", then your configuration ....

Reviewer: No, it is fine as is.

>   pluginJs.configs.recommended,

Reviewer: As we have `pluginJs.configs.recommended` in the configuration example then can you also update the `js.configs.recommended` by replacing `js` in following para?

Author: you mean ``pluginJs`` instead of ``js`` ?  I thought they are different :thinking: ...

Reviewer: Yes replacing `js` with `pluginJs`, its just a variable name.

`import pluginJs from "@eslint/js";`

Author: I see . Thanks for the guidance .
